src/vivarium_gates_iv_iron/data/loader.py

```diff
@@ -27,6 +27,14 @@
 
 from vivarium_gates_iv_iron.constants import data_keys
 
+ARTIFACT_INDEX_COLUMNS = [
+    'location',
+    'sex',
+    'age_start',
+    'age_end',
+    'year_start',
+    'year_end'
+]
```

Comment:
I have this in `metadata.py` so it can be accessed if needed by model code, not just data-loading code.